arelle/ModelDocument.py

```diff
@@ -1007,10 +1012,15 @@ def schemaDiscover(self, rootElement, isIncluded, isSupplemental, namespace):
                     namespace=namespace, targetNamespace=targetNamespace)
             if (self.modelXbrl.modelManager.validateDisclosureSystem and
                 self.modelXbrl.modelManager.disclosureSystem.disallowedHrefOfNamespace(self.uri, targetNamespace)):
-                    self.modelXbrl.error(("EFM.6.22.02", "GFM.1.1.3", "SBR.NL.2.1.0.06" if self.uri.startswith("http") else "SBR.NL.2.2.0.17"),
```

Comment:
This check is for disallowed taxonomy references. In my opinion I think that this is an incorrect version of GFM 1.1.3.

The GFM manual specificially calls out that `the URI content of the xlink:href attribute, the xsi:schemaLocation attribute and the schemaLocation attribute must be relative and contain no forward slashes, or a recognized external location of a standard taxonomy schema file, or a "#" followed by a shorthand xpointer.`

We should be checking the composition of the `hrefs` not whether they are valid or not.